components/projects/projectmodel.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+// Define the schema
+const { Schema } = mongoose;
+
+const ProjectSchema = new Schema({
+  title: String,
```

Comment:
Currently a project can be saved even if there is no information provided to the model. I think at least the `title` and the `authorId` should be required fields if a project is to be saved.

Reply:
It does? Oh, good catch, I'll look into that.

Reply:
I see what's going on. This issue doesn't deal with saving the project at all, it's all about the GET. The POST will be handled with issue #46 but I'll make the fields required now.